contracts/Strategy.sol

```diff
@@ -285,30 +304,57 @@ contract Strategy is BaseStrategy {
         }
     }
 
-    function collectTradingFees() internal {
+    function claimRewards() external isVaultManager {
```

Comment:
We don't need this function anymore. We have `claimAndSellRewards(False)` which is the same.